packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -468,6 +475,28 @@ export default class Editor {
         return this.defaultFormat;
     }
 
+    /**
+     * For internal use only
+     */
+    public getTempDivForPaste(): HTMLElement {
+        if (!this.pasteDiv || !this.pasteDiv.parentNode) {
+            this.pasteDiv = fromHtml(
+                CONTAINER_HTML,
+                this.getDocument()
+            )[0] as HTMLElement;
+            this.insertNode(this.pasteDiv, {
+                position: ContentPosition.Outside,
+                updateCursor: false,
+                replaceSelection: false,
+                insertOnNewLine: false,
+            });
+        } else {
+            this.pasteDiv.style.display = '';
```

Comment:
Is it necessary to set display to '' ?

Reply:
Yes. This is necessary.
Each time after use, I set display to none. So I need to set it back to '' to use again.